### ISSUE TITLE
BUG: Link doesn't work on root level route group

### ISSUE THREAD

**Issue by arolson101** [ProNextJS/declarative-routing#29]
Links to the root don't work when they're in a route group.  The react component ends up with an href=""
```ts
export const Dashboard = makeRoute(
  "/(dashboard)",
  {
    ...defaultInfo,
    ...DashboardRoute.Route
  }
);
```
I think the "/(dashboard) gets removed, resulting in an empty href link instead of href="/".  If I manually change it to "/" or "/(dashboard)/" then it works.

This is using 0.1.15

**Comment by jherr** [comment]
So you have the `/` route in a route group? `/(dashboard)/page.tsx`?

**Comment by arolson101** [comment]
Correct

**Comment by jherr** [comment]
Should be fixed in 0.1.16. You can copy the `makeRoute.tsx` from the project and replace your existing file.
